docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -154,6 +154,32 @@ for instance:
     - if you have extended `CountryGridFactory`, revise your changes because class changed its namespace
     - if you have extended `CountryFormType`, revise your changes – new fields are available
     - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added
+- Microservices has been removed and their funcionality has been moved to framework ((#793)[https://github.com/shopsys/shopsys/pull/793/]):
+    - classes from microservices has been moved to `shopsys/framework` package under `Product\Search` namespace
+    - definitions of indexes has been moved to project folder `src/Shopsys/ShopBundle/Resources/definition`, copy them to your project as well
+        - add `shopsys.elasticsearch.structure_dir: '%shopsys.resources_dir%/definition/'` to your `paths.yml` config file
+    - Symfony commands for Elasticsearch management has changed their namespace from `shopsys:microservice:product-search` to `shopsys:product-search`
+        - update your `build.xml` and `build-dev.xml` to reflect this change
+        - if you have extended those command in your project, update their class and file names and `$defaultName` property appropriately
+    - `StructureManager` has been moved to `packages/framework/src/Component/Elasticsearch` and renamed to `ElasticsearchStructureManager`
```

Comment:
This isnt UPGRADE instruction, there should be something like `If you are extending the StructureManager, change its name or FQN.

Reply:
Well yeah I have problem with this, because that is something to be written every time we rename something and I expect that our community is intelligent enough to know, that if they use such class, they have to do something :)

I have added some instruction at the end of the line, but I am not happy about that